docker/sagemaker/serve

```diff
@@ -134,4 +150,4 @@ elif [ "${is_mme_mode}" = false ]; then
     SAGEMAKER_ARGS="${SAGEMAKER_ARGS} --load-model=${SAGEMAKER_TRITON_DEFAULT_MODEL_NAME}"
 fi
 
-tritonserver --allow-sagemaker=true --allow-grpc=false --allow-http=false --allow-metrics=false --model-control-mode=explicit $SAGEMAKER_ARGS
+tritonserver --allow-sagemaker=true --allow-grpc=true --allow-http=false --allow-metrics=true --model-control-mode=explicit $SAGEMAKER_ARGS
```

Comment:
You need to expose way to configure the grpc and metrics port because the default value may not comply with SageMaker safe range if specified, this [test case](https://github.com/triton-inference-server/server/blob/main/qa/L0_sagemaker/test.sh#L229-L243) is an example that `serve` script will fail because default ports are not in safe range.

Reply:
@GuanLuo can you review again, thank you